packages/stripe/src/payment-method.component.ts

```diff
@@ -12,7 +12,9 @@ import * as angular from 'angular'
 import 'angular-material'
 
 // Services
+// TODO: We don't need to force the stripe import here if the StripeService below is used
```

Comment:
I've found `StripeService` only imports the Type used by typescript only. it didn't actually load the @stratusjs/stripe/stripe file, which is required when compiled

Reply:
That makes sense.  It would need a qualified JavaScript usage for it to end up in the compilation.